.github/workflows/master.yml

```diff
@@ -63,6 +63,8 @@ jobs:
       if: steps.conversion_lock.outcome == 'success' && ! env.CI_SKIP
       shell: bash -el {0}
       run: |
+          test -f broken/example.txt
+          test -f unbroken/example.txt
```

Comment:
These should be in the PR workflow. Moving them over in PR ( https://github.com/conda-forge/admin-requests/pull/613 )